content/en/docs/concepts/security/secrets-good-practices.md

```diff
@@ -0,0 +1,95 @@
+---
+reviewers:
+title: Good practices for Kubernetes Secrets
+description: >
+  Principles and practices for good Secret management for cluster administrators and application developers.
+content_type: concept
```

Comment:
We can add a `weight` to this page

Reply:
I don't think I properly understand how weights work, does the weight of adjacent pages also need to change to reorder in the TOC?

Reply:
Explicit weight would be desirable. By default, Hugo sort pages by name.